sagemathinc/cocalc
# webapp: adding rel="noopener" to external links

# Description
Basically https://developers.google.com/web/tools/lighthouse/audits/noopener

# Testing Steps
* this should be pretty safe to merge

what I didn't think about when I made this needs review:
* shift-click an editor tab to open in a small window
* open video chat window (that should definitely run independent of cocalc)

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
